app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -0,0 +1,48 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.dto.AuthenticationCreateData;
+import com.codesoom.assignment.dto.AuthenticationResultData;
+import com.codesoom.assignment.dto.SessionResultData;
+import com.codesoom.assignment.util.JwtUtil;
+import io.jsonwebtoken.Claims;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+
+/** 사용자 인증에 대한 요청을 수행한다. */
+@Service
+@Transactional
+public class AuthenticationService {
+    private final JwtUtil jwtUtil;
+
+    public AuthenticationService(JwtUtil jwtUtil) {
+        this.jwtUtil = jwtUtil;
+    }
+
+    /**
+     * 주어진 사용자를 검증하고 토큰 문자열을 생성하여 리턴한다.
+     *
+     * @param authenticationCreateData - 검증하고자 하는 사용자
+     * @return 주어진 사용자를 이용하여 생성된 토큰 문자열
+     */
+    public SessionResultData createToken(AuthenticationCreateData authenticationCreateData) {
+        String accessToken = jwtUtil.encode(
+                authenticationCreateData.getEmail()
+                ,authenticationCreateData.getPassword()
```

Comment:
```suggestion
                authenticationCreateData.getEmail(),
                authenticationCreateData.getPassword()
```

comma가 위에 들어가는게 더 자연스러울 것 같습니다 ㅎㅎ Golang에서는 comma를 위와 같이 맨 끝에 작성하지 않으면 애초에 컴파일이 안됩니다.

![image](https://user-images.githubusercontent.com/14071105/110131174-3abbe880-7e0d-11eb-8b9c-63d83a6bbabc.png)

Reply:
아하~ 네, 어디에다가 쓰면 좋을지 계속 고민되었는데 수정하겠습니다~